packages/apps-config/src/ui/logos/index.ts

```diff
@@ -115,6 +115,8 @@ import nodeBasilisk from './nodes/basilisk.png';
 import nodeBeast from './nodes/beast.svg';
 import nodeBifrost from './nodes/bifrost.svg';
 import nodeBitCountry from './nodes/bitcountry.png';
+import nodeBridgeHubBlack from './nodes/BridgeHubBlack.svg';
+import nodeBridgeHub from './nodes/nodeBridgeHub.svg';
```

Comment:
```suggestion
```

Unused as per https://github.com/polkadot-js/apps/actions/runs/4007320742/jobs/6880461427#step:3:327

Reply:
True. It will be used by the Polkadot bridge hub later. Removed it for the moment.